system-addon/test/unit/utils.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * GlobalOverrider - Utility that allows you to override properties on the global object.
+ *                   Seee unit-entry.js for example usage.
+ */
+class GlobalOverrider {
+  constructor() {
+    this.originalGlobals = new Map();
+    this.addedGlobals = new Set();
+    this.sandbox = sinon.sandbox.create();
+  }
+  _override(key, value) {
```

Comment:
I think each of the methods in this class want a bit of JSDoc, so that it's possible to easily tell how they should be used.